barretenberg/cpp/src/barretenberg/ultra_honk/decider_verifier.cpp

```diff
@@ -65,12 +67,13 @@ template <typename Flavor> bool DeciderVerifier_<Flavor>::verify()
         return false;
     }
     bool consistency_checked = true;
+    ClaimBatcher claim_batcher{
+        .unshifted = ClaimBatch{ commitments.get_unshifted(), sumcheck_output.claimed_evaluations.get_unshifted() },
+        .shifted = ClaimBatch{ commitments.get_to_be_shifted(), sumcheck_output.claimed_evaluations.get_shifted() }
```

Comment:
can't you do
```
.shifted = {commitments.get_to_be_shifted(), sumcheck_output.claimed_evaluations.get_shifted()}
```
does the `ClaimBatch` struct type need to be specified?

Reply:
I thought so too but it doesn't work in this case bc the parameters are std::optional. It seems like I could avoid it by explicitly initializing each input as optional but it ends up looking much worse